src/parser/pegen.ts

```diff
@@ -576,6 +577,18 @@ export function dummy_name(p: Parser): Name {
 //     return cache;
 // }
 
+export function get_keyword_or_name_type(p: Parser, name: string): number {
+    assert(p.keywords !== null);
+
+    if (p.keywords.has(name)) {
+        // typescript doesn't seem to know about Map's :D sorry about the ts-ignore
```

Comment:
I don't know if this is a typescript or a deno fail but we should probably open a ticket in the right repo this feels a bit dumb